changelog/_template.md.jinja2

```diff
@@ -1,10 +1,10 @@
 {# Based on https://github.com/hawkowl/towncrier/blob/master/src/towncrier/templates/default.rst #}
-{% for section in sections %}{% if section %}{{section}}{% endif %}{% if sections[section] %}{% for category, val in definitions.items() if category in sections[section] %}
+{% if top_line %}{{ top_line }} {{ top_underline * ((top_line)|length)}} {% elif versiondata.name %}{{ versiondata.name }} {{ versiondata.version }} ({{ versiondata.date }}) {{ top_underline * ((versiondata.name + versiondata.version + versiondata.date)|length + 4)}}{% else %}{{ versiondata.version }} ({{ versiondata.date }}) {{ top_underline * ((versiondata.version + versiondata.date)|length + 3)}}{% endif %}{% for section in sections %}{% if section %}{{section}}{% endif %}{% if sections[section] %}{% for category, val in definitions.items() if category in sections[section] %}
 
 {{ "### " + definitions[category]['name'] }}
 {% if definitions[category]['showcontent'] %}{% for text, values in sections[section][category]|dictsort(by='value') %}{% set issue_joiner = joiner(', ') %}- {% for value in values|sort %}{{ issue_joiner() }}{{ value }}{% endfor %}: {{ text }}
 {% endfor %}{% else %}- {{ sections[section][category]['']|sort|join(', ') }}{% endif %}{% if sections[section][category]|length == 0 %} No significant changes.
 
 {% else %}{% endif %}{% endfor %}{% else %} No significant changes.
 
-{% endif %}{% endfor %}
\ No newline at end of file
+{% endif %}{% endfor %}
```

Comment:
the template changed, I found the solution in [this comment](https://github.com/twisted/towncrier/issues/340#issuecomment-819059654)